### DISCUSSION TITLE
how to extract cloud point files XYZ ?

### DISCUSSION THREAD

**Discussion by krenskiBR** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#253] [Q&A] [answered]
hello, im want to extract the cloud points files generate from the lidar,  to open in CAD or cloud point softwares, anybody know how to do it?

**Comment by PiotrMachowski** [comment, marked as answer]
There is no such data in the cloud. Map is stored as an already processed image
